ui/packages/catalog-ui-search/src/main/webapp/component/upload-batch-item/upload-batch-item.view.js

```diff
@@ -35,9 +35,7 @@ module.exports = Marionette.LayoutView.extend({
   regions: {
     uploadDetails: '> .upload-details .details-summary',
   },
-  initialize() {
-    const modelJSON = this.model.toJSON()
-  },
+  initialize() {},
```

Comment:
✏️ I believe you can (and probably should) get rid of this default override since it's doing nothing now.